src/components/album/ArtistName.css

```diff
@@ -0,0 +1,24 @@
+.artist{
+  color:#a0a0a0;
+  font-size: 14px;
+  margin-top: 1px;
+  height: auto;
+  border: 1px white sold;
+  font-family: Helvetica;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.artist:hover{
+  text-decoration: underline;
+  color: #ffffff;
+}
+
+.artist:not(:last-of-type):after {
+  content: ", ";
+  
+}
+
+.artist:nth-last-of-type(2):after {
+  content: " & ";
+}
```

Comment:
Nice! Could you think of a way to not include the "," or "&" in the actual link? 👀 

![Skärmavbild 2024-03-13 kl  16 01 10](https://github.com/Technigo/project-music-releases-vite/assets/51529201/cd1f51b1-f6e0-4891-8c8d-f158aeb25d41)